metaflow/metaflow_config.py

```diff
@@ -43,7 +43,22 @@
 DEFAULT_GCP_CLIENT_PROVIDER = from_conf("DEFAULT_GCP_CLIENT_PROVIDER", "gcp-default")
 DEFAULT_SECRETS_BACKEND_TYPE = from_conf("DEFAULT_SECRETS_BACKEND_TYPE")
 DEFAULT_SECRETS_ROLE = from_conf("DEFAULT_SECRETS_ROLE")
-
+# configuration for Linter/Validators
+FLOW_VALIDATORS = from_conf("FLOW_VALIDATORS", {
```

Comment:
to be thought through, implemented and vetted a 'means' to transform a cli arg into such a 'dictionary/setting' @tuulos